### PR TITLE
Add Swift 5.7 CI

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

File: docker/docker-compose.2004.57.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: vscode-swift:20.04-5.7
+    build:
+      args:
+        base_image: "swiftlang/swift:nightly-5.7-focal"
```
Comment (tomerd) [inline docker/docker-compose.2004.57.yaml:9]: do we want to use the release one now?

Comment (adam-fowler) [reply docker/docker-compose.2004.57.yaml:9]: Yes, just realised we are using the 5.6 nightly as well.  Will fix both

**Comment by tomerd** [comment]
@swift-server-bot test this please

**Review by tomerd — APPROVED** [review]

**Comment by tomerd** [comment]
@swift-server-bot test this please

**Pull request merged**